packages/typeless-sample-bot/test/fixtures/deleteSchema.ts

```diff
@@ -0,0 +1,55 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * This application demonstrates how to perform basic operations on
+ * schemas with the Google Cloud Pub/Sub API.
+ *
+ * For more information, see the README.md under /pubsub and the documentation
+ * at https://cloud.google.com/pubsub/docs.
+ */
+
+// sample-metadata:
+//   title: Delete a previously created schema
```

Comment:
Small bug I noticed, looking at the snapshot, the spacing gets lost in front of title:

```
// sample-metadata:
// title: Delete a previously created schema
// description: Deletes a schema which was previously created in the project.
// usage: node deleteSchema.js <schema-name-or-id>
```

It may be worth looking at the output to see if this is the actual behavior.

Reply:
Ah... yeah, that's not great. That's probably an actual bug.

There's an extra set of work I need to do anyway in regards to adding `'use strict'` and a `generated` comment into output files, so maybe I could do this as part of that work.